### ISSUE TITLE
Thank you!

### ISSUE THREAD

**Issue by xttweaponttx** [ElishaAz/Sayboard#61]
I'm not sure how to send positive feedback, so I'm opening this issue to say THANKS! This app works so dang well, I'm very happy such a great solution exists! Cheers!

**Comment by Charles7z** [comment]
I agree. I just installed futo voice input a few days ago and it was nearly 500 megabytes installed. While it worked very well there were still issues where you needed a punctuation keyboard and an editing keyboard like sayboard has. So now i just have to get used to using voice input again since i stopped using it for so long because google was so bad. -- Done with say board! 😁
